src/mpi/init/init_async.c

```diff
@@ -47,6 +47,20 @@
         progress threads of second local process - on "2,3".
         Cannot work together with MPIR_CVAR_NUM_CLIQUES or MPIR_CVAR_ODD_EVEN_CLIQUES.
 
+    - name        : MPIR_CVAR_CH4_PROGRESS_THREAD_AUTO_AFFINITY
```

Comment:
We don't need another CVAR. When user set `MPIR_CVAR_CH4_PROGRESS_THREAD_AFFINITY=on` or `yes|ON|YES`, threat it as auto.

Reply:
How about use the value "auto" instead of "yes" or "on"? Then options will be "auto|no|<numbers>"? I think this can help avoid some confusion.

Reply:
Sure

Reply:
Done.